src/MICore/Debugger.cs

```diff
@@ -55,6 +55,22 @@ public bool IsClosed
             }
         }
 
+        public bool IsDebuggerRunning
+        {
+            get
+            {
+                if (_localDebuggerPid > 0)
+                {
+                    if (PlatformUtilities.IsLinux() || PlatformUtilities.IsOSX())
+                    {
+                        return UnixNativeMethods.Kill(_localDebuggerPid, 0) == 0;
+                    }
+                }
+
+                return false;
+            }
+        }
```

Comment:
might want to comment that this call to kill will not actually kill the process because 0 is being passed, this through me off for a second.